### PR TITLE
Release 0.7.2

### PR DESCRIPTION
**Pull request by kasparsd**
Minor release.

Diff: https://github.com/WordPress/two-factor/compare/0.7.1...13c30418aa54640f739f59b23ed23897b76b4caf

## Changelog

- Security improvement: Store the second factor authentication step nonce hashed to prevent leaking it via database read access [#453](https://github.com/WordPress/two-factor/pull/453). Props to [@calvinalkan](https://github.com/calvinalkan) for reporting the issue.
- Fix: Add `wp_specialchars_decode()` to escape the HTML entity on the Email Subject line ([#412](https://github.com/WordPress/two-factor/issues/412)), props [@nbwpuk](https://github.com/nbwpuk).
- Fix: Use `hash_equals()` when comparing the email token ([#425](https://github.com/WordPress/two-factor/issues/425)), props [@Mati02K](https://github.com/Mati02K).
- Tooling: Introduce `@wordpress/env` for development tooling and move to GitHub actions for CI ([#436](https://github.com/WordPress/two-factor/pull/436)).

### CODE REVIEW

**Review by georgestephanis — APPROVED** [review]
Everything already pushed looked reasonable, so should be fine.

File: readme.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Contributors: georgestephanis, valendesigns, stevenkword, extendwings, sgrant, aaroncampbell, johnbillion, stevegrunwell, netweb, kasparsd, alihusnainarshad, passoniate
 Tags: two factor, two step, authentication, login, totp, fido u2f, u2f, email, backup codes, 2fa, yubikey
 Requires at least: 4.3
-Tested up to: 5.9
+Tested up to: 6.0
 Requires PHP: 5.6
 Stable tag: trunk
```
Comment (jeffpaul) [inline readme.txt:7]: do we want to update the `Stable tag` to 0.7.2 as well?

**Review by jeffpaul — APPROVED** [review]

**Comment by jeffpaul** [comment]
@kasparsd do you want to also land #438 ahead of releasing 0.7.2 to automate that release step?

File: readme.txt (hunk 1 above)
Comment (kasparsd) [reply readme.txt:7]: It is using `trunk` for some historic reason. I don't know why or what would happen if we change that :)

**Pull request merged**